BiweeklyMeetings/2022-05-10.md

```diff
@@ -0,0 +1,493 @@
+
+**Matthias**-:
+Yeah. Many of them are draft or just starts of discussions. And maybe they are fine to just leave us as discussions until they make it as a pull request. But then we still need to do some form of issue managements at least to close the old discussions and, do a bit of house cleaning. I don't think we will go through the whole thread of discussions on that one just here. And it hasn't got any updates for about six months also so I'm not sure if the discussion has settled or not in the end. That's one we need to go through and read, and see what's up, but right now it just seems like an idea that sounds to have been abandoned, but I'm not sure.
+
+**M. Àngels**-:
+Shall we ping the author and...
+
+**Matthias**-:
+Well, we should read, read the conversation thread first and then see if it's... I would say we have two outcome already, either we ping the other to see if it's still relevant based on what we've read in the thread, or we just close the thread completely, close the issue from being, resolved and installed. And if they ever want to come back to it, they're free to reopen the issue and... Or make a pull request actually acting on it. But first I would like to get a sense of what's being said in the thread.
+
+**M. Àngels**-:
+Do you want to add to the agenda in two weeks to check if you have time to read it or to go and read it, or do you want to it a month?
+
+**Matthias**-:
+I would say for now we don't add it to the agenda. I will read it. And if it's something that we want to add to the agenda in the sense that the discussion hasn't been settled, then yeah. We'll just add it. If not, then I will close the issue with a comment explaining exactly that.
+
+**M. Àngels**-:
+Okay.
+
+**Matthias**-:
+Yes, indeed. And as is saying in the chat, there is a lot of controversy over whether you should be able to add royalties to a project which did not need them within the original policy. And this is indeed a topic that's been discussed in the past here. So I think this is part of what the thread will be about. People disagreeing about whether or not a CIP makes sense in itself, but it's also two separate concerns in a way. That a CIP in the end is just a proposal always. And whether or not these proposals end up being adopted by marketplaces and projects, that's a different topic. So if the proposal makes sense from a technical standpoint and it's something that could indeed be implemented and not lead to security issues or stuff like that, that makes for valid proposal. And you may still disagree with the proposal and not implement it, that doesn't discard the proposal from being proposed. But if there is indeed no proposals, then we can just close it. Maybe for just a discussions, like have an idea. What do you think? Bad idea. Okay. Close it.
+
+**M. Àngels**-:
+Do you make any comment on what Matthias just commented or shall we move forward?
+
+**Matthias**-:
+Only to agree. 
```

Comment:
my guess is that Sebastien would have said this (?)

Reply:
Casually agreeing with myself :joy: